tests/console/firewalld.pm

```diff
@@ -18,23 +18,22 @@ use testapi;
 use utils qw(systemctl zypper_call);
 use version_utils qw(is_sle is_leap is_tumbleweed);
 
+sub uses_iptables {
+    return is_sle('<15-SP3') || is_leap('<15.3');
+}
+
 # Check Service State, enable it if necessary, set default zone to public
 sub pre_test {
     zypper_call('in firewalld');
     zypper_call('info firewalld');
     record_info 'Check Service State';
-    assert_script_run("if ! systemctl is-active -q firewalld; then systemctl start firewalld; fi");
+    script_run('echo "FIREWALLD_ARGS=--debug" >> /etc/sysconfig/firewalld');
```

Comment:
Does systemd's `EnvironmentFile` support overwriting like this? I suggest using sed or awk instead

Reply:
> Does systemd's `EnvironmentFile` support overwriting like this? I suggest using sed or awk instead

Yeah, it does phobos.qa.suse.de/tests/3857474 :) at least no complains from openQA run, and enabling it on my machine: https://asciinema.org/a/JqxqBglPBNV3HLnX0UQYbOajG

Reply:
Found it in the systemd.exec man page as well:

> If the same variable is set twice from these files, the files will be read in the order they are specified and the later setting will override the earlier setting.

Though I'm not sure whether that's about several files or several statements in the same file...